gst/nnstreamer/include/nnstreamer_plugin_api_trainer.h

```diff
@@ -51,7 +52,11 @@ typedef struct _GstTensorTrainerFrameworkInfo
 {
   const char *name;    /**< Name of the neural network framework, searchable by FRAMEWORK property. */
   unsigned int is_training_complete;  /**< Check if training is complete, Use unsigned int instead of gboolean because this is refered by custom plugins. */
-  unsigned int epoch_cnt;    /**< Number of currently completed epochs */
+  unsigned int epoch_count;    /**< Number of currently completed epochs */
+  double training_loss;    /**< Training loss of the model being trained in the subplugin */
+  double training_accuracy;    /**< Training accuracy of the model being trained in the subplugin */
+  double validation_loss;    /**< Validation loss of the model being trained in the subplugin */
+  double validation_accuracy;    /**< Validation accuracy of the model being trained in the subplugin */
```

Comment:
This cannot be "FrameworkInfo".

Assume that there are multiple ```tensor_trainer``` instances in a single pipeline.

Reply:
Thank you.